app/code/Magento/Config/Model/Config.php

```diff
@@ -287,13 +302,13 @@ private function getField(string $sectionId, string $groupId, string $fieldId):
     /**
      * Get field path
      *
-     * @param Field $field
+     * @param FieldElement $field
      * @param string $fieldId Need for support of clone_field feature
      * @param array $oldConfig Need for compatibility with _processGroup()
      * @param array $extraOldGroups Need for compatibility with _processGroup()
      * @return string
      */
-    private function getFieldPath(Field $field, string $fieldId, array &$oldConfig, array &$extraOldGroups): string
+    private function getFieldPath(FieldElement $field, string $fieldId, array &$oldConfig, array &$extraOldGroups): string // phpcs:ignore Generic.Files.LineLength
```

Comment:
Let's not ignore the coding standards.

```suggestion
    private function getFieldPath(
        FieldElement $field,
        string $fieldId,
        array &$oldConfig,
        array &$extraOldGroups
    ): string
```

Reply:
Have made changes as you suggested.